### ISSUE TITLE
Feature request: visual note selector

### ISSUE THREAD

**Issue by allyunion** [isalin/TBbard#20]
One of the things I noticed that the program handles poorly is when multiple notes are played in the same second.  As we aren't able to reproduce that, some of the selections of the notes are not optimal.  It would be nice to have the ability to remove bad sounding notes and save the results (after post processing) so you can load processed songs (instead of MIDI files)

Additionally, saving/loader the text files would be better for later replays

**Comment by isalin** [comment]
Yeah, I agree! There should be a system for saving/loading songs. It would also help make it much faster to switching back and fourth between songs. 

I'll definitely look into adding that in the next version! Unfortunately, it might take a while, since I'll most likely be a bit busy in real life for a while.

**Comment by DAOWAce** [comment]
👍  I would like this as well.

Currently the only way to do things is create our own text files and copy/paste everything every time.  Works, if slow and clunky.
